### DISCUSSION TITLE
Use a dot for instead of an uppercase to call a component?

### DISCUSSION THREAD

**Discussion by jpsca** [jpsca/jinjax#65] [General]
I just discovered a similar library for Phoenix (an Elixir web framework) called Phoenix Core Components.

Instead of JinjaX’s “start with an uppercase letter”, they use a dot to differentiate between components calls and html tags.

![IMG_2708](https://github.com/jpsca/jinjax/assets/67524204/f828e9b9-0848-4267-970c-1969c33b3175)

I like it… maybe is something that jinjaX could also support. Which one do you prefer, `<GradientText>…</GradientText>` or `<.gradient_text>…</.gradient_text>`?

**Comment by CiberNin** [comment]
I prefer the capitalization. I is more inline with JSX style syntax which inspired the library.
